Tests/selectItem.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class selectItem extends TestCase
```

Comment:
I'd suggest renamign this class (and file) to be `selectItemTest` just to follow naming conventions